rust/rpmostree-client/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "rpmostree-client"
```

Comment:
Bikeshed: `rpm-ostree-client`? To match the project and binary name we're wrapping.

Reply:
It's interesting since I didn't think much about the name when writing this, I just typed it out and then continued.  And I found myself hesitating changing it and this made me think about *why* that is.  Why did I chose "rpmostree" and not "rpm-ostree"?  And why am I hesitating to change?

I think the answer is that to me the hyphen is best when the words involved have "equal binding" - and that's not the case here.  Here it's really "(rpm-ostree) - client" as you and I know.  But someone else might equally parse it as "rpm - (ostree-client)" or even just "rpm - ostree - client".

If one knows what "rpm-ostree" is of course then it's more obvious...but still.  I realized I've been using this naming scheme elsewhere; most obviously in the source file naming (`src/app/rpmostree-builtin-kargs.cxx`) as well as C symbols `rpmostree_option_context_parse()`.

Now perhaps to start the project should have just been named "rpmostree" but...eh ;)

So, I dunno.  I'm not strictly against changing this; we're certainly not consistent in this respect.